package.json

```diff
@@ -1,74 +1,89 @@
 {
-  "name": "charta",
-  "version": "0.0.1",
+  "name": "@dharmaprotocol/contracts",
+  "version": "0.0.14",
+  "main": "dist/artifacts/index.js",
+  "typings": "dist/types/artifacts/index.d.ts",
+  "files": [
+    "build",
+    "dist",
+    "artifacts",
+    "contracts",
+    "migrations",
+    "scripts",
+    "tsconfig.dist.json",
+    "truffle.js"
+  ],
   "description": "New & improved smart contracts for Dharma protocol",
   "scripts": {
+    "deploy:development": "bash scripts/deploy_development.sh",
+    "deploy:kovan": "bash scripts/deploy_kovan.sh",
+    "dist": "bash scripts/prepare_dist.sh",
     "generate-typings": "abi-gen --abis './build/contracts/*.json' --out './types/generated' --template './types/contract_templates/contract.mustache' --partials './types/contract_templates/partials/*.mustache'",
     "transpile": "rm -rf ./transpiled; copyfiles ./build/**/* ./transpiled; tsc",
     "lint-ts": "tslint migrations/*.ts test/**/*.ts",
     "lint-sol": "solhint contracts/*.sol contracts/libraries/*.sol contracts/examples/*.sol contracts/test/**/.sol",
     "lint": "npm run lint-ts; npm run lint-sol",
     "compile": "truffle compile",
-    "soltsice": "soltsice",
-    "migrate": "npm run transpile; truffle migrate",
-    "prettify": "prettier --write test/ts/**/*.ts types/**/*.ts types/*.ts",
+    "migrate": "truffle migrate",
+    "prettify": "prettier --write test/ts/**/*.ts types/**/*.ts types/*.ts artifacts/*.ts",
     "test": "npm run compile; npm run generate-typings; npm run migrate; truffle test",
     "chain": "ganache-cli --networkId 70 --accounts 20",
     "validate": "npm ls"
   },
   "repository": "git@github.com:dharmaprotocol/charta.git",
   "author": "Nadav Hollander <nadav@dharma.io>",
   "license": "ISC",
-  "dependencies": {
```

Comment:
Looks like we'll want to keep `NonFungibleToken` and `zeppelin-solidity` as deps

Reply:
Done.  Merging.